build/images/tf_operator/Dockerfile

```diff
@@ -1,9 +1,14 @@
 # TODO(jlewi): How can we make it work with golang:1.8.2-alpine
 FROM golang:1.8.2
 
+# TODO(jlewi): We should probably change the directory to /opt/kubeflow.
 RUN mkdir -p /opt/mlkube
 RUN mkdir -p /opt/mlkube/test
 RUN mkdir -p /opt/tensorflow_k8s/dashboard/
+RUN mkdir -p /opt/kubeflow/samples
+
+COPY tf_smoke.py /opt/kubeflow/samples/
+RUN chmod a+x /opt/kubeflow/samples/*
```

Comment:
I think we should keep the operator image clean and small, but it is not in high priority, the code works for me.

Reply:
That's a good point. I think when our release infra is more mature and its easier to build multiple images that will make sense.

Practically speaking the sample added in this PR takes up no space. The biggest space is probably nodejs for the UI vs. operator.

Reply:
SGTM, I will file an issue and we just keep it in the PR

Reply:
Actually, I am thinking if we should separate the UI and the operator. Maybe we should place the UI in an independent repo.